Examples/generate-app.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Copyright (c) 2015-present, Microsoft Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+# The goal of this script is to automate the testing of react-native, react-native-code-push and 
+# their internal compatibility. Run **./run.sh** to create an app and test it. 
+
+# You can configure below variables in the bash. 
+
+# 1. `code_push_version`
+# 2. `react_native_code_push_version`
+# 3. `react_native_version`
+# 4. `mobile_env`
+
+# If you use other libs or packages, and fail to run this test, welcome to modify this script 
+# so that we can reproduce any issues that you have.
+
+
+echo 'Automate synchronization testing between React Native and React Native Code Push';
```

Comment:
I think this line would be better modified to describe the script as a CodePush + RN sample app generation script